RecoEgamma/EgammaElectronProducers/plugins/GsfElectronProducer.cc

```diff
@@ -91,3 +113,138 @@ void GsfElectronProducer::checkPfTranslatorParameters( edm::ParameterSet const &
      }
    }
  }
+
+
+// now deprecated
+void GsfElectronProducer::addPflowInfo(reco::GsfElectronCollection & electrons, edm::Event const& event) const
+{
+  //Isolation Value Maps for PF and EcalDriven electrons
+  typedef std::vector< edm::Handle< edm::ValueMap<double> > > IsolationValueMaps;
+  IsolationValueMaps pfIsolationValues;
+  IsolationValueMaps edIsolationValues;
+
+  //Fill in the Isolation Value Maps for PF and EcalDriven electrons
+  std::vector<edm::InputTag> inputTagIsoVals;
+  if(! inputCfg_.pfIsoVals.empty() ) {
+    inputTagIsoVals.push_back(inputCfg_.pfIsoVals.getParameter<edm::InputTag>("pfSumChargedHadronPt"));
+    inputTagIsoVals.push_back(inputCfg_.pfIsoVals.getParameter<edm::InputTag>("pfSumPhotonEt"));
+    inputTagIsoVals.push_back(inputCfg_.pfIsoVals.getParameter<edm::InputTag>("pfSumNeutralHadronEt"));
+
+    pfIsolationValues.resize(inputTagIsoVals.size());
+
+    for (size_t j = 0; j<inputTagIsoVals.size(); ++j) {
+      event.getByLabel(inputTagIsoVals[j], pfIsolationValues[j]);
+    }
+
+  }
+
+  if(! inputCfg_.edIsoVals.empty() ) {
+    inputTagIsoVals.clear();
+    inputTagIsoVals.push_back(inputCfg_.edIsoVals.getParameter<edm::InputTag>("edSumChargedHadronPt"));
+    inputTagIsoVals.push_back(inputCfg_.edIsoVals.getParameter<edm::InputTag>("edSumPhotonEt"));
+    inputTagIsoVals.push_back(inputCfg_.edIsoVals.getParameter<edm::InputTag>("edSumNeutralHadronEt"));
+
+    edIsolationValues.resize(inputTagIsoVals.size());
+
+    for (size_t j = 0; j<inputTagIsoVals.size(); ++j) {
+      event.getByLabel(inputTagIsoVals[j], edIsolationValues[j]);
+    }
+  }
+
+  bool found ;
+  auto edElectrons = event.getHandle(inputCfg_.previousGsfElectrons);
+  auto pfElectrons = event.getHandle(inputCfg_.pflowGsfElectronsTag);
+  reco::GsfElectronCollection::const_iterator pfElectron, edElectron ;
+  unsigned int edIndex, pfIndex ;
+
+  for(auto& el : electrons)
+   {
+
+    // Retreive info from pflow electrons
+    found = false ;
+    for
+     ( pfIndex = 0, pfElectron = pfElectrons->begin() ; pfElectron != pfElectrons->end() ; pfIndex++, pfElectron++ )
+     {
+      if (pfElectron->gsfTrack() == el.gsfTrack())
+       {
+        if (found)
+         {
+          edm::LogWarning("GsfElectronProducer")<<"associated pfGsfElectron already found" ;
+         }
+        else
+         {
+          found = true ;
+
+        // Isolation Values
```

Comment:
Indent as above

Reply:
Thanks for the comments! This indentation is always an issue... What do you think, should I just use the .clang-format form let's say https://github.com/cms-sw/cmssw/pull/25893 to put the blocks of code that I move elsewhere in a good shape? That would be simple for us but has the drawback that there will be inconsistent code style within a file. What do you thin?